docs/source/datasets.rst

```diff
@@ -0,0 +1,72 @@
+Datasets
+========================
+To perform an automated systematic review, ASReview requires a dataset of papers.
+The dataset of papers usually results from an online library search for all studies related to the topic to review.
+All papers in the dataset need to be screened, with the goal to select all relevant papers. 
+
+For testing and demonstrating ASReview, the software offers `three built-in datasets <#built-in-demonstration-datasets>`__
+It is also possible to perform an automated systematic review on `your own dataset <#using-your-own-data>`__.
+Every paper in the dataset carries metadata such as title, abstract, author, et cetera. 
+
+Built-in demonstration datasets
+---------------------
```

Comment:
This one gives issues with rendering. The underline should equal (or exceed) the length of the title. Same length is preferred.